.github/workflows/patrol-prepare.yaml

```diff
@@ -155,13 +155,13 @@ jobs:
       - name: Run unit tests
         working-directory: packages/patrol/example/ios
         run: |
-          xcodebuild test \
+          set -o pipefail && xcodebuild test \
```

Comment:
any reason it is not a separate line?

Reply:
copied from `xcbeautify`'s docs, that's why haha